spec/features/navbar_spec.rb

```diff
@@ -45,4 +45,19 @@ def toggle_profile_dropdown
       page.execute_script('document.getElementById("navbarProfileDropdown").click()')
     end
   end
+
+  describe 'public page' do
+    before do
+      sign_out user
+      visit root_path
+    end
+
+    it 'does not contain a link to the notes page' do
```

Comment:
It was not a functional requirement. Actually, showing the home button would not harm either because the home controller displays the login page unless signed in.